BUILD.md

```diff
@@ -44,10 +44,10 @@ process. For instance:
     cd debug/src/unity
     make -j 4
 
-will perform up to 4 build tasks in parallel. When building in release/ mode,
+will perform up to 4 build tasks in parallel. When building in `release/` mode,
 Turi Create does require a large amount of memory to compile with the
 heaviest toolkit requiring 1GB of RAM. Where K is the amount of memory you
-have on your machine in GB, we recommend not exceeding make -j K
+have on your machine in GB, we recommend not exceeding make `-j` K
```

Comment:
"make" and "K" are part of the command. I think it should be: `make -j K`

Reply:
that's ok